x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_export_by_object_ids.test.ts

```diff
@@ -174,23 +173,6 @@ describe('get_export_by_object_ids', () => {
       expect(exports).toEqual(expected);
     });
 
-    test('it returns error when readRules throws error', async () => {
```

Comment:
With the change to retrieve all rules with a single query, errors are simply propagated up to the route where they're caught and a `500` is returned in the API response.